markdown/articles/2020/12/20/historyofdoomsday.md

```diff
@@ -0,0 +1,1030 @@
+storm player's arsenal. The deck also ran the full 4 Sensei's Tops as these functioned as additional card filtering, 
+long game advantage and often as an 8th card in hand during your combo turn or as a way to gain extra draws using a trick
+with Top and Chain of Vapor to bounce it in response to the trigger.
+
+The deck was very open to intepretation and, in Emidln's own words:
+
+> "This archetype is extremely customizable. The basic requirements are blue/black fetchlands, brainstorm, dark ritual, 
+> lion's eye diamond, and tendrils of agony. Everything else is basically up in the air."
+> - [emidln](http://www.mtgthesource.com/forums/showthread.php?7800-Deck-Fetchland-Tendrils&p=183642&viewfull=1#post183642)
+
+Overall the decks were starting to become a lot more streamlined. With the onset of better cantrips, exploration of correct
+cantripping and fetching mechanics were starting to be explored. What are now assumed basic mechanics for a lot of Legacy decks were
+almost unheard of back then as the format, and the skills involved, were still fledging. This is also where we start to see the
+birth of more modern storm variants with the 'Xerox' concepts of cantrips over lands really taking effect. 4 Brainstorm, 4 Top, 4
+Ponder and 4 Street Wraith became a staple build with Wraith allowing you to grab any card fetched from Mystical Tutor in the same
+turn as a *build your own {{Vampiric Tutor}}* type affair. 
+
+It was also around this time that Emidln and Wastedlife set up the stormboards. BZK was a team affiliated site with other people
+who worked on seperate teams not wanting to share testing results or tech. Because of this, Emidln wanted to set up a non-team
+affiliated space where people who loved all sorts of Storm combo could discuss things freely. Here he was later joined by other
+active users including prominent french player Lejay. 
+
+Within this new forum, and with other online spaces becoming more active, players were able to really develop the fundamentals of
+Legacy Storm. It is also around this era that we see start to revisit an old friend and give it a new feel.
+
+
+## Doomsday Meet Tendrils. Legacy Meet DDFT
+
+It was was online user Breathweapon who first suggested taking the Doomsday shells that were floating around and try incorporating
+the traditional storm kill into it. As mentioned with the Nausea deck above, Doomsday was a potential tool that people could
+run already in storm as a non-graveyard dependant engine away from ill-Gotten Gains but it was still never the main focus of the decks.
+Breathweapon suggested a few Storm based piles, most very simplistic variants such as:
+
+Cantrip + {B}{B}{B}{B}
+<row variant="pile">{{!Cruel Bargain}} {{!DR}} {{!DR}} {{!SDT}} {{!ToA}}</row>
+
+On seeing  this Emidln sat down online with goobafish (inventor of Canadian Threshold), desolutionist and birdman to
```

Comment:
I don't know the right number of people to call out, but goobafish (David Caplan), desolutionist (Shawn Anthony), birdman (Kyle Cappello), rsaunders (Chris Saunders), and Whit3_Ghost (Will Eldridge) were all BZK teammates who helped test hundreds of matches with all of us making suggestions/improvements. Without access to the BZK forums, "correct" attribution is impossible. It was a team effort.